recipe_scrapers/primaledgehealth.py

```diff
@@ -20,7 +21,22 @@ def image(self):
         return self.schema.image()
 
     def ingredients(self):
-        return self.schema.ingredients()
+        ingredients_li = self.soup.select(".wprm-recipe-ingredient")
+        ingredients_list = [
+            normalize_string(
+                li.find("span", {"class": "wprm-recipe-ingredient-amount"}).get_text()
+            ).replace("\u00C2", "")
+            + " "
+            + normalize_string(
+                li.find("span", {"class": "wprm-recipe-ingredient-unit"}).get_text()
+            ).replace("\u00C2", "")
+            + " "
+            + normalize_string(
+                li.find("span", {"class": "wprm-recipe-ingredient-name"}).get_text()
+            ).replace("\u00C2", "")
+            for li in ingredients_li
+        ]
```

Comment:
This code could do with some refactoring I think.  I'll experiment with it locally soon to see what's possible; feel free to try some adjustments yourself too.

My guess is that some use of list comprehensions could make this a bit neater.